BINDINGS.md

```diff
@@ -201,9 +201,22 @@ Then the following controls should be empty:
 
 ## List Binding
 
-* Check that list contains given values: 
-[Given("(.*?) (contain|not contain) the following items:")]
-[Then("(.*?) should (contain|contain in exact order|not contain) the following items:")]
+* Check that list has given values
+[Given("(.+?) (has|does not have) the following items:")]
+[Then("(.+?) should (have|not have) the following items:")]
+Example:
+Then "Categories" should have the following items:
+
+* Check that list has given values in exact order
+[Then("(.+?) should have in exact order the following items:")]
+Example:
+Then "Categories" should have in exact order the following items:
+
+* Check that list contains given values
+[Given("(.+?) (has|does not have) the following items:")]
+[Then("(.+?) should (have|not have) the following items:")]
```

Comment:
wrong description, [Given("(.+?) (has|does not have) the following items:")] -> [Given("(.+?) (contains|does not contain) the following items:")] ?